src/kirby/directives/theme-color/theme-color.directive.ts

```diff
@@ -8,7 +8,8 @@ import { ColorHelper } from '../../helpers/color-helper';
   selector: `kirby-card[themeColor],
              kirby-badge[themeColor],
              kirby-icon[themeColor],
-             kirby-toolbar[themeColor]`,
+             kirby-toolbar[themeColor],
+             ion-item[themeColor]`,
```

Comment:
Is this a leftover?

I don't see you use the `themeColor` directive on an `ion-item`?

Reply:
It was, thanks!